src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -255,12 +341,25 @@ public void setBranch(String branch) {
         authorConfig.setBranch(branch);
     }
 
-    public void updateDisplayName(String branch) {
-        this.displayName = displayName.substring(0, displayName.lastIndexOf('[') + 1) + branch + "]";
+    /**
+     * Replaces the branch parameter in {@code displayName} with {@code branch}.
+     */
+    private void updateDisplayName(String branch) {
+        int uniqueIdIndex = displayName.lastIndexOf(' ');
+        String uniqueId = (uniqueIdIndex == -1) ? "" : displayName.substring(uniqueIdIndex);
```

Comment:
The same code is used in `RepoConfiguration#updateOutputFolderName`

Can refactor this into another common method.

Reply:
Oops, forgot to remove those